wls-exporter-core/src/main/java/com/oracle/wls/exporter/domain/ExporterConfig.java

```diff
@@ -24,6 +24,10 @@
  * @author Russell Gold
  */
 public class ExporterConfig implements MetricsProcessor {
+    // Due to a bug in the WLS REST API, a query that includes this field will fail if not run with admin privileges.
+    // The code thus ensures that it is excluded from all queries.
+    private static final String[] FORBIDDEN_FIELDS = {"JDBCServiceRuntime:JDBCDataSourceRuntimeMBeans:properties"};
```

Comment:
Just curious... Is this the only affected field or just the only one that we know about?

Reply:
It's the only one we've found so far. But I have written the exclusion code to make it easy to add others if we find them.